### PR TITLE
core: contact aliases

### PR DESCRIPTION
**Pull request by jr-simplex**

### CODE REVIEW

File: src/Simplex/Chat.hs [hunk 1]
```diff
@@ -2575,6 +2583,8 @@ chatCommandP =
       "/_call status @" *> (APICallStatus <$> A.decimal <* A.space <*> strP),
       "/_call get" $> APIGetCallInvitations,
       "/_profile " *> (APIUpdateProfile <$> jsonP),
+      "/_set alias @" *> (APISetContactAlias <$> A.decimal <* A.space <*> textP),
```
Comment (epoberezkin) [inline src/Simplex/Chat.hs:2586]:
```suggestion
      "/_set alias @" *> (APISetContactAlias <$> A.decimal <*> (A.space *> textP <|> pure "")),
```

File: src/Simplex/Chat/Migrations/M20220824_profiles_user_alias.hs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+module Simplex.Chat.Migrations.M20220824_profiles_user_alias where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20220824_profiles_user_alias :: Query
+m20220824_profiles_user_alias =
+  [sql|
+ALTER TABLE contact_profiles ADD COLUMN user_alias TEXT;
```
Comment (epoberezkin) [inline src/Simplex/Chat/Migrations/M20220824_profiles_user_alias.hs:11]: CHECK, UPDATE

File: src/Simplex/Chat.hs [hunk 3]
```diff
@@ -578,8 +578,11 @@ processChatCommand = \case
     withCurrentCall contactId $ \userId ct call ->
       updateCallItemStatus userId ct call receivedStatus Nothing $> Just call
   APIUpdateProfile profile -> withUser (`updateProfile` profile)
-  APISetContactAlias contactId userAlias -> pure CRCmdOk
-  APIUnsetContactAlias contactId -> pure CRCmdOk
+  APISetContactAlias contactId userAlias -> withUser $ \User {userId} -> do
+    ct' <- withStore $ \db -> do
+      ct <- getContact db userId contactId
+      liftIO $ updateContactProfileUserAlias db userId ct userAlias
```
Comment (epoberezkin) [inline src/Simplex/Chat.hs:584]: This is not user alias - it’s contact alias

Comment (epoberezkin) [inline src/Simplex/Chat.hs:584]:
```suggestion
      liftIO $ updateContactAlias db userId ct userAlias
```

File: src/Simplex/Chat.hs [hunk 4]
```diff
@@ -578,8 +578,11 @@ processChatCommand = \case
...
+    pure $ CRContactProfileUserAliasUpdated ct'
```
Comment (epoberezkin) [inline src/Simplex/Chat.hs:585]:
```suggestion
    pure $ CRContactAliasUpdated ct'
```

File: src/Simplex/Chat.hs (hunk 3 above)
Comment (jr-simplex) [reply src/Simplex/Chat.hs:584]: it's alias given by the user - to distinguish other fields in profile that are defined by contact

**Pull request merged**